app/views/logs_searches/contact.html.erb

```diff
@@ -1,8 +1,16 @@
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
+<%= render "layouts/form_errors", resource: @search %>
+
+<div class='govuk-grid-row'>
+  <div class='govuk-grid-column-two-thirds'>
+    <%= link_to "Back to search options", new_logs_search_path, class: "govuk-back-link" %>
+  </div>
+
+  <h1 class='govuk-heading-l govuk-grid-column-full'>Search for user details</h1>
+
+  <div class='govuk-grid-column-two-thirds'>
     <%= form_with url: super_admin_wifi_user_search_path do |f| %>
       <p>
-        <%= f.label :search_term, 'Username or contact details', class: 'govuk-label', for: :search_term %>
+        <%= f.label :search_term, 'Username, email address or phone number', class: 'govuk-label', for: :search_term %>
```

Comment:
it's a bit academic, but isn't it missing IP address?

Reply:
The original feature didn't include finding a user by IP address, not sure if we have this on a per user basis though I know we do have MAC address. It's worth considering as an enhancement though.